src/main/java/frc/robot/vision/multivisionsources/MultiVisionSources.java

```diff
@@ -0,0 +1,84 @@
+package frc.robot.vision.multivisionsources;
+
+import frc.robot.poseestimator.observations.VisionRobotPoseObservation;
+import frc.robot.subsystems.GBSubsystem;
+import frc.robot.vision.VisionConstants;
+import frc.robot.vision.rawdata.RawVisionData;
+import frc.robot.vision.sources.VisionSource;
+import org.littletonrobotics.junction.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class MultiVisionSources<T extends VisionSource<? extends RawVisionData>> extends GBSubsystem {
```

Comment:
maybe make RawVisionData interface, so it solve u all the funny T extends

Reply:
No

Reply:
why?

Reply:
@GilStein1 you reverted that change when I've done it..